db/seeds.js

```diff
@@ -0,0 +1,18 @@
+const db = connect("mongodb://localhost:27017/anoninjaDB");
+
+db.posts.drop();
+
+db.posts.insertMany([
+    {
+        title: "Ninjas",
+        pseudonym: "Ninja",
+        content: "Ninjas are better than Pirates",
+        date: new Date(),
+    },
+    {
+        title: "Pirates",
+        pseudonym: "Pirate",
+        content: "Pirates are better than ninjas",
+        date: new Date(),
+    },
+]);
```

Comment:
Seeding looks good and works as it should, is cool to think your storing the Date() type, as opposed to a string, so you can manipulate it later 👍